java-client/src/main/java/org/opensearch/client/opensearch/core/TermvectorsResponse.java

```diff
@@ -157,8 +157,10 @@ protected void serializeInternal(JsonGenerator generator, JsonpMapper mapper) {
 		generator.writeKey("took");
 		generator.write(this.took);
 
-		generator.writeKey("_version");
-		generator.write(this.version);
+        if (null != this.version) {
```

Comment:
Is this optional? Maybe add a test?

Reply:
You have right. According to the server sources https://github.com/opensearch-project/OpenSearch/blob/7dc6683838951e1a211c6ff634d9eea93e2be3c2/server/src/main/java/org/opensearch/action/termvectors/TermVectorsResponse.java#L205 _version is still delivered.